src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -62,8 +62,9 @@ trait Read {
 
   /** Is this read mapped? */
   final val isMapped: Boolean = this match {
-    case r: MappedRead   => true
-    case r: UnmappedRead => false
+    case r: MappedRead       => true
```

Comment:
Is there a reason to do it this way vs. implementing `isMapped` in each subclass? (that would be the more OO approach)